File: eMRTDReader/sdk/src/main/java/com/example/emrtdreader/domain/PassportReadResult.java
package com.example.emrtdreader.domain;

import com.example.emrtdreader.models.PassportChipData;
import com.example.emrtdreader.models.VerificationResult;

import java.io.Serializable;

public class PassportReadResult implements Serializable {
    public final PassportChipData chipData;
    public final VerificationResult verification;

    public PassportReadResult(PassportChipData chipData, VerificationResult verification) {
        this.chipData = chipData;
        this.verification = verification;
    }
}

File: eMRTDReader/app/src/main/java/com/example/emrtdreader/domain/PassportReadResult.kt
package com.example.emrtdreader.domain

import com.example.emrtdreader.crypto.AuthResult
import com.example.emrtdreader.data.PassportData

data class PassportReadResult(
    val passportData: PassportData,
    val authResult: AuthResult,
    val json: String
)
